build.gradle.kts

```diff
@@ -55,21 +55,15 @@ apply(plugin = "kover")
 dependencies {
     implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
     implementation(tools.bundles.kotlin)
-    implementation("com.fasterxml.jackson.module:jackson-module-afterburner")
```

Comment:
Why we remove afterburnera and not Jacksona as well?

Reply:
because jackson is used  in for example GemUser

Reply:
but that module of jason that speedup serialization, idk i we use in service template we also dont wont to take that